src/hotspot/share/opto/node.cpp

```diff
@@ -664,55 +664,51 @@ void Node::destruct(PhaseValues* phase) {
   }
 }
 
-//------------------------------grow-------------------------------------------
-// Grow the input array, making space for more edges
-void Node::grow(uint len) {
+//------------------------------resize_array-------------------------------------------
+ // Resize input or output array to grow it the next larger power-of-2 bigger
+// than len.
+void Node::resize_array( Node**& array, node_idx_t& max_size, uint len, bool is_in) {
   Arena* arena = Compile::current()->node_arena();
-  uint new_max = _max;
+  uint new_max = max_size;
   if( new_max == 0 ) {
-    _max = 4;
-    _in = (Node**)arena->Amalloc(4*sizeof(Node*));
-    Node** to = _in;
-    to[0] = nullptr;
-    to[1] = nullptr;
-    to[2] = nullptr;
-    to[3] = nullptr;
-    return;
+    max_size = 4;
+    array = (Node**)arena->Amalloc(4*sizeof(Node*));
```

Comment:
```suggestion
    array = (Node**)arena->Amalloc(4 * sizeof(Node*));
```
We generally want to have spaces around operators.

See:
https://github.com/openjdk/jdk/blob/master/doc/hotspot-style.md

> Use spaces around operators, especially comparisons and assignments. (Relaxable for boolean expressions and high-precedence operators in classic math-style formulas.)